tests/unit/models/search.js

```diff
@@ -0,0 +1,287 @@
+        "title": "New England Journal of Medicine reconsiders relationship with industry",
+        "date": "2015-05-12",
+        "authors": "McCarthy, M.",
+        "inPress": false,
+        "availableThroughBrowzine": true,
+        "startPage": "h2575",
+        "endPage": "h2575",
+        "browzineWebLink": "https://browzine.com/libraries/XXX/journals/18126/issues/7764583?showArticleInContext=doi:10.1136/bmj.h2575"
+      },
+      "included": [{
+        "id": 18126,
+        "type": "journals",
+        "title": "theBMJ",
+        "issn": "09598138",
+        "sjrValue": 2.567,
+        "coverImageUrl": "https://assets.thirdiron.com/images/covers/0959-8138.png",
+        "browzineEnabled": true,
+        "browzineWebLink": "https://develop.browzine.com/libraries/XXX/journals/18126"
+      }]
+    };
+  });
+
+  afterEach(function() {
+
+  });
+
+  it("search model should exist", function() {
+    expect(search).toBeDefined();
+  });
+
+  it("should not enhance a search result without a document", function() {
+    var scope = {};
+
+    expect(search.shouldEnhance(scope)).toEqual(false);
+  });
+
+  it("should enhance a journal search result with an issn", function() {
+    var scope = {
+      document: {
+        content_type: "Journal",
+        issns: ["0028-4793"]
+      }
+    };
+
+    expect(search.shouldEnhance(scope)).toEqual(true);
+  });
+
+  it("should enhance an eJournal search result with an issn", function() {
+    var scope = {
+      document: {
+        content_type: "eJournal",
+        issns: ["0028-479X"]
+      }
+    };
+
+    expect(search.shouldEnhance(scope)).toEqual(true);
+  });
+
+  it("should enhance an article search result with a doi", function() {
+    var scope = {
+      document: {
+        content_type: "Journal Article",
+        dois: ["10.1136/bmj.h2575"]
+      }
+    };
+
+    expect(search.shouldEnhance(scope)).toEqual(true);
+  });
+
+  it("should not enhance a journal search result without an issn or eissn", function() {
+    var scope = {
+      document: {
+        content_type: "Journal"
+      }
+    };
+
+    expect(search.shouldEnhance(scope)).toEqual(false);
+  });
+
+  it("should not enhance an article search result without a doi", function() {
+    var scope = {
+      document: {
+        content_type: "Journal Article"
+      }
+    };
+
+    expect(search.shouldEnhance(scope)).toEqual(false);
+  });
+
+  it("should not enhance a journal search result without a content type", function() {
+    var scope = {
+      document: {
+        issns: ["0028-4793"]
+      }
+    };
+
+    expect(search.shouldEnhance(scope)).toEqual(false);
+  });
+
+  it("should not enhance an article search result without a content type", function() {
+    var scope = {
+      document: {
+        dois: ["10.1136/bmj.h2575"]
+      }
+    };
+
+    expect(search.shouldEnhance(scope)).toEqual(false);
+  });
+
+  it("should build a journal endpoint for a journal search result", function() {
```

Comment:
In general, grouping the tests by method they exercise might make this easier to quickly navigate

Reply:
This should help w/ organization, I like it. Thanks!